### PR TITLE
[LLM] vLLM: Add Preempt for scheduler

### PR DESCRIPTION
**Pull request by xiangyuT**
## Description

Implement Preempt_by_recompute method for vllm.

### CODE REVIEW

File: python/llm/src/bigdl/llm/vllm/core/scheduler.py [hunk 1]
```diff
@@ -101,6 +117,8 @@ def __init__(
         self.cleaned: List[int] = []
         self.kv_cache = kv_cache
         # Co(gc): We no longer have the swapped space as we are not deciding which to swap
```
Comment (gc-fu) [inline python/llm/src/bigdl/llm/vllm/core/scheduler.py:119]: We may want to delete this comment since we now having a swap area

Comment (xiangyuT) [reply python/llm/src/bigdl/llm/vllm/core/scheduler.py:119]: Currently maybe not. We have only implement recompute right now and swap is still unavailable.

File: python/llm/src/bigdl/llm/vllm/core/scheduler.py [hunk 2]
```diff
@@ -101,6 +117,8 @@ def __init__(
...
+        self.swapped: List[SequenceGroup] = []
+        self.seq_ability = 8
```
[the remark this answers is not in the document]
Comment (xiangyuT) [reply python/llm/src/bigdl/llm/vllm/core/scheduler.py:121]: Of course. Maybe just use max_num_seq?

[the remark this answers is not in the document]
Comment (xiangyuT) [reply python/llm/src/bigdl/llm/vllm/core/scheduler.py:121]: > Consider adding this to scheduler config later?

Of course. Maybe just use max_num_seq?

**Pull request merged**